velox/docs/functions/spark/datetime.rst

```diff
@@ -13,6 +13,16 @@ These functions support TIMESTAMP and DATE input types.
     If num_days is a negative value then these amount of days will be
     deducted from start_date.
 
+.. spark:function:: datediff(endDate, startDate) -> integer
+    Returns the number of days from startDate to endDate. ::
+
+    SELECT datediff('2009-07-31', '2009-07-30'); -- 1
+    SELECT datediff('2009-07-30', '2009-07-31'); -- -1
+
+.. spark:function:: date_diff(start, end) -> date
```

Comment:
from the code is seems like the parameters are exactly the same as above. Are they the same or in inverse order in Spark? If they are the same, maybe just have the comment saying this is an alias for "datediff"

Reply:
They are the same, I updated the doc, just say this is an alias for "datediff".